docs/tutorials/getting_started/command_line.rst

```diff
@@ -38,9 +37,9 @@ Here are some of the plots that are output when running with the latest total nu
 
 The results from the tournament itself (ordered by median score):
 
-.. image:: http://axelrod-python.github.io/tournament/assets/strategies_boxplot.svg
-   :width: 50%
-   :align: center
```

Comment:
Why did this throw a warning? Is it considered bad practice to have images linked to from outside of the docs?